src/auth/index.js

```diff
@@ -20,7 +20,6 @@
  *   BASE_URL,
  *   LMS_BASE_URL,
  *   LOGIN_URL,
- *   LOGIN_URL,
```

Comment:
Ah, this is supposed to be `LOGOUT_URL`, along with line 34 below...

Reply:
Line 34 should read:
```
 *   logoutUrl: LOGOUT_URL,
```

Reply:
Fixed